packit_service/worker/build/build_helper.py

```diff
@@ -91,6 +97,47 @@ def local_project(self) -> LocalProject:
             )
         return self._local_project
 
+    @property
+    def is_gitlab_instance(self) -> bool:
+        if self._is_gitlab_instance is None:
+            self._is_gitlab_instance = isinstance(self.project, GitlabProject)
+
+        return self._is_gitlab_instance
```

Comment:
I am not sure if this helps too much. I am leaving that to your judgement if it is clearer now. (=You don't need to change that if you want..;)